Israel National Newspaper Library.js

```diff
@@ -0,0 +1,134 @@
+{
+	"translatorID": "9d8099b7-1c50-4159-9a67-f9fc1fb3b463",
+	"label": "Israel National Newspaper Library",
+	"creator": "Anonymus",
+	"target": "^https://www\\.nli\\.org\\.il/",
+	"minVersion": "6.0",
+	"maxVersion": "",
+	"priority": 100,
+	"inRepository": true,
+	"translatorType": 4,
+	"browserSupport": "gcsibv",
+	"lastUpdated": "2025-10-27 11:48:08"
+}
+
+/*
+	***** BEGIN LICENSE BLOCK *****
+
+	Made by Zotero contributors.
+	
+	This file is part of Zotero.
+	
+	Zotero is free software: you can redistribute it and/or modify
+	it under the terms of the GNU Affero General Public License as published by
+	the Free Software Foundation, either version 3 of the License, or
+	(at your option) any later version.
+
+	Zotero is distributed in the hope that it will be useful,
+	but WITHOUT ANY WARRANTY; without even the implied warranty of
+	MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+	GNU Affero General Public License for more details.
+
+	You should have received a copy of the GNU Affero General Public License
+	along with Zotero. If not, see <http://www.gnu.org/licenses/>.
+
+	***** END LICENSE BLOCK *****
+*/
+
+function detectWeb(doc, url) {
+	if (url.includes("/newspapers/") && (url.includes("/article/") || url.includes("/page/"))) {
+		return "newspaperArticle";
+	}
+	return false;
+}
+
+async function doWeb(doc, url) {
+	await scrape(doc, url);
+}
+
+async function scrape(doc, url) {
+	const item = new Zotero.Item("newspaperArticle");
+
+	// Get JSON-LD data
+	let jsonLDData = null;
```

Comment:
Can'r unresolve the previous thread, so posting one more time - the earlier fix for this one got reverted, please double-check.